packages/sdk/src/utils/utils.js

```diff
@@ -7,7 +7,7 @@ export const parseUri = uri => {
 
   return {
     pathname: parser.pathname,
-    search: qs.parse(parser.search),
+    search: decode(parser.search.substring(1)),
```

Comment:
This is needed otherwise the object has `{ '?foo': 'bar, ... }` which is not bad but not what we want here. Same as with the hashbang.